### ISSUE TITLE
Run validation correction script  on several languages

### ISSUE THREAD

**Issue by Adrijaned** [Common-Voice/sentence-collector#259]
#225 changed record approval logic, however, sentences that had just two approving votes on them cast before merging of that PR, are still marked as invalid in kinto. Would it be thus possible to run the script introduced in #250 on all less active languages to bring them up to date? I am aware of this problem existing in at least Czech dataset (locale cs)

**Comment by MichaelKohler** [comment]
As this is not an issue for the export I'm marking this as P2 and will do it once I have time. How did you end up with not approved sentences though? When I wrote the script I tested against all languages and didn't see any others that were affected. Are you using any other interface apart from the official Sentence Collector for cs?

**Comment by Adrijaned** [comment]
None that I am aware of. I reviewed like 5 sentences from my android app just to test that thing works, but I am pretty certain that I was the first reviewer on those, so that shouldn't have any impact at it. On closer look though, it appears most of the sentences around the one I noticed this at are marked correctly, which might make it just a random quirk? I was looking at the particular sentence just because it's screenshot was posted in the mozilla.cz's telegram, and noticed it not being approved just accidentally, The specific sentence in question is https://kinto.mozvoice.org/v1/buckets/App/collections/Sentences_Meta_cs/records/e4a75d01a066bc9d26923b3fac0011f6b02d6ef25e07202a393feb079939ddde

Could this be just a random one-off quirk?
